src/modules/aws_vpc/mappers/tags.ts

```diff
@@ -21,3 +21,22 @@ export async function updateTags(client: EC2, resourceId: string, tags?: { [key:
     Tags: tgs,
   });
 }
+
+export function convertTagsFromAws(inputTags?: Tag[]) {
```

Comment:
is that different from other tags for other entities in AWS?

Reply:
No it's the same. Do we have a function for it?

Reply:
We have a similar pattern on ec2, so it could be reused